scripts/cibuild.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+set -e
+
+vagrant up --provision
+vagrant ssh -c "
+    cd /vagrant
+    echo -e '\nRunning linter...'
+
+    # run linter with console output
+    npm run lint
```

Comment:
Chiming in to note that `scripts/cibuild` generally does call `scripts/test`. I normally prefer having all test content in `scripts/test` so that it can be run in development without also running any additional steps in `cibuild`.